Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -2783,6 +2903,8 @@ internal static async Task MakeBucket_Test1(MinioClient minio)
         finally
         {
             await minio.RemoveBucketAsync(rbArgs).ConfigureAwait(false);
+            // Wait for cleanup to be complete before running the next test
+            Thread.Sleep(1500);
```

Comment:
Why is this needed? Can't we use a 'sync' option?

Reply:
They will all be removed as part of the Thread.Sleep cleanup task.